_includes/tutorial-content.html

```diff
@@ -48,95 +43,102 @@ <h4 class="subtitle" id="{{ step.title | slugify }}">
       <div class="columns">
         <div class="column is-full">
           <h2 class="title tutorial-section-title" id="test-it">Test it</h2>
-          {% for step in site.data.harnesses[page.static_data][page.stack].test.steps %}
+          {% for step in
+          site.data.harnesses[page.static_data][page.stack].test.steps %}
 
           <div class="tutorial-try-it-step content">
-            <h4 class="subtitle" id="{{ step.title | slugify }}">
+            <h2
+              class="title tutorial-section-item-title"
+              id="{{ step.title | slugify }}"
+            >
+              {{ step.title }}
+            </h2>
+
+            <div class="content-item">
               <div class="num">{{ forloop.index }}</div>
-              <div class="text">{{ step.title }}</div>
-            </h4>
-
-            {% for section in step.content %}
-
-            {% if section.render.skip != true %}
-
-            {% capture prose %}{% include {{ section.render.file }} %}{% endcapture %}
-            {{ prose | asciidocify }}
-
-            {% endif %}
-
-            {% endfor %}
+              <div class="item-display">
+                {% for section in step.content %} {% if section.render.skip !=
+                true %} {% capture prose %}{% include {{ section.render.file }}
```

Comment:
same

Reply:
cc: @scott-confluent is working on this in another PR